dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -145,21 +167,22 @@ export const recordPerformance = async (): Promise<void> => {
 		redirectCount: performanceAPI.navigation.redirectCount,
 	};
 
-	const ophan = await getOphan();
+	const ophan = await getOphan(renderingTarget);
 	ophan.record({
 		performance,
 	});
 };
 
 const experiencesSet = new Set(['dotcom-rendering']);
 export const recordExperiences = async (
+	renderingTarget: RenderingTarget,
```

Comment:
I think this new signature is quite dangerous and confusion, I think it would be best to have experiences be an explicit array of strings.

```ts
export const recordExperiences = async (
  renderingTarget: RenderingTarget,
  experiences: string[], // no longer a rest argument
): Promise<void> => { 
  /* do things */
}
```

Reply:
Think I addressed this, not sure why GH can't tell! Let me know if not!